recipe/build.sh

```diff
@@ -1,13 +1,5 @@
 #!/bin/bash
 
-# FIXME: This is a hack to make sure the environment is activated.
-# The reason this is required is due to the conda-build issue
-# mentioned below.
-#
-# https://github.com/conda/conda-build/issues/910
-#
-# source activate "${CONDA_DEFAULT_ENV}"
-
 export CXXFLAGS="$CXXFLAGS -std=c++14"
```

Comment:
I think this is not necessary anymore, since ``conda-forge`` is using gcc 7

Reply:
I had an error with the c++1z standard, that's why I added that

Reply:
error was:

```
vhdl_element.hh:49:36: error: ISO C++1z does not allow dynamic exception specifications
    void* operator new(size_t size) throw (std::bad_alloc);
                                    ^~~~~
```

see here: https://circleci.com/gh/conda-forge/iverilog-feedstock/23?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Ok then... but this is deprecated since c++11

Reply:
AFAIU, in c++14 it is still only deprecated (_i.e._ does not raise an error), so I thought it was ok to step back to 14 rather than 11.